src/main/app/client_native.cljs

```diff
@@ -2,7 +2,7 @@
   "Entry point for native client."
   (:require
     [app.application :refer [SPA]]
-    [com.fulcrologic.fulcro-native.expo-application :as expo]
```

Comment:
I don't understand. Why didn't you just fix the old ns???

Reply:
I wanted the template to be usable right away. 
I will issue an update to fulcro-native repo!